README.md

```diff
@@ -1 +1,169 @@
-# catalog-of-my-things
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [catalog-of-my-things] <a name="catalog-of-my-things"></a>
+
+> **[catalog-of-my-things]** is an app that can add Add a book, List all books, List all labels, List all music albums, List all genres, Add a music album, List of games, List all authors, Add a game. It's our capstone project for the Ruby muodule built with ruby. 
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Language</summary>
+  <ul>
+    <li>Ruby</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[add_a_book,List_all_books,List_all_labels]**
+- **[List_all_music_albums,List_all_genres,Add_a_music_album]**
+- **[List_of_games,List_all_authors,Add_a_game]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+<!-- ## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://google.com) -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Code Editor
+- Internet Connection
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```git clone https://github.com/EvansSnave/catalog-of-my-things.git```
+
+
+### Install
+
+Install this project with:
+
+```bundle install```
+
+
+### Usage
+
+To run the project, execute the following command:
+
+``` ruby main.rb```
+
+### Run tests
+
+To run tests, run the following command:
+
+```rspec spec```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Author1**
```

Comment:
- [ ] [OPTIONAL] These should be your names in order to make readers know the names of all contributors.

Reply:
We will work on that!